silkworm/capi/common.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+   Copyright 2024 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "common.hpp"
+
+#include <cstring>
+
+std::filesystem::path make_path(const char data_dir_path[SILKWORM_PATH_SIZE]) {
```

Comment:
The name `make` suggests that it creates path on the disk
```suggestion
std::filesystem::path cast_path(const char data_dir_path[SILKWORM_PATH_SIZE]) {
```

Reply:
You're right, thank you. I'm going to rename it `parse_path` to follow the naming of other functions we already have here, e.g. [parse_cors_domains](https://github.com/erigontech/silkworm/blob/capi_rpcdaemon_settings/silkworm/capi/rpcdaemon.cpp#L60) [parse_peer_urls](https://github.com/erigontech/silkworm/blob/capi_rpcdaemon_settings/silkworm/capi/sentry.cpp#L40)